src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -78,6 +78,62 @@ public static RefKind GetRefKind(this BoundExpression node)
             }
         }
 
+        /// <summary>
+        /// Returns the RefKind of an expression
+        /// if the expression represents a symbol, the RefKind of the symbol is returned
+        /// if the expression is a "by-ref" expression like conditional operator or InlineArrayAccess the RefKind is calculated and returned
+        /// otherwise this function returns RefKind.None 
+        /// </summary>
+        public static RefKind GetRefKindEx(this BoundExpression node)
```

Comment:
I suggest to move this helper to SpillSequenceSpiller.cs right after ```VisitConditionalOperator```, it doesn't have to be an extension method.